setup/02-robot_syntax_rule.py

```diff
@@ -1,25 +1,68 @@
-from rflint.common import SuiteRule, TestRule, ERROR
+from rflint.common import SuiteRule, TestRule, ERROR, WARNING
 
 MUST_KEYWORDS = [
         "Open Browser",
         "Input Text",
-        "Page Should Contain"
+        "Input Password"
     ]
 
+LOGIN_KEYWORDS = [
+        "Click Element",
+        "Click Button"
+]
+
+VERIFY_KEYWORDS = [
+        "Page Should Contain",
+        "Page Should Contain Element",
+        "Title Should Be",
+        "Wait Until Page Contains",
+        "Wait Until Page Contains Element"
+]
+
 class TestCaseImplementation02(TestRule):
     severity = ERROR
 
     def apply(self, test):
+        default_message = "Did you find all keywords from seleniumlibrary documentation?"
         report = False
         test_steps = []
         for step in test.steps:
             if len(step) > 1:
-                test_steps.append(step[1])
+                test_steps.append(step[1].lower())
         for keyword in MUST_KEYWORDS:
-            if not keyword.strip() in test_steps:
+            if not keyword.strip().lower() in test_steps:
```

Comment:
This has the problem of littering the same nomalization, which often leads to forgetting it somewhere and thus to bugs. I would introduce a method here:

```python
def normalize(self, kw):
     return kw.strip().title()

# ...
test_steps.append(self.normalize(step[1]))
# ...
if not self.normalize(keyword) in test_steps:
# ...
for keyword in test_steps:
    if self.normalize(keyword) in LOGIN_KEYWORDS:
# ...
for keyword in test_steps:
    if self.normalize(keyword) in VERIFY_KEYWORDS:
```